pages/resources/[page].js

```diff
@@ -62,6 +69,26 @@ function Resources() {
     paid: paid || '',
   };
 
+  const handleLogin = value => loginUser(value);
+
+  const handleLoginSuccess = ({ token }) => {
+    setAuthCookies({ token });
+  };
+
+  const handleVote = (/* voteDirection, updateVoteCountFunc */) => {
```

Comment:
Looks like this is a bit unfinished?

Reply:
I was gonna leave the actual implementation of this function for someone else but I guess I can finish it up.

Reply:
I could open up an issue if you don't want to do it! That's totally fair since this page isn't considered finished.

Reply:
actually, I would like to take this on if you don't mind. I could use some additional PR